.github/workflows/CI.yml

```diff
@@ -13,13 +13,9 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - "1.6"
           - "1"
-          - "nightly"
```

Comment:
This should definitely still build "nightly", since you'd want to catch any issues introduced in the latest julia.